x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardFollowNodeTask.java

```diff
@@ -90,6 +358,12 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             {
                 builder.field(PROCESSED_GLOBAL_CHECKPOINT_FIELD.getPreferredName(), processedGlobalCheckpoint);
             }
+            {
```

Comment:
nit  - what do we need these extra `{}` - just inline this in the previous block?

Reply:
I started doing this here, because I agreed with this comment: https://github.com/elastic/x-pack-elasticsearch/pull/4290#discussion_r179411015

Reply:
one set of `{}` is awesome. But you can inline all fields in one set?

Reply:
yes, that makes sense.